### PR TITLE
Making it pretty

### PR DESCRIPTION
**Pull request by AlexBuchko**
Changed the css files to apply a nice dark color scheme to website, changed font, and touched up spacing

image
image
image

### CODE REVIEW

**Comment by jnlao** [comment]
you included an empty file in the pull request

File: frontend/src/AboutPage.js [hunk 1]
```diff
@@ -9,9 +9,10 @@ function AboutPage() {
```
Comment (jnlao) [inline frontend/src/AboutPage.js:9]: in file above, "About.css" there is no code or content. can delete this

**Review by jnlao — CHANGES_REQUESTED** [review]

File: frontend/src/AboutPage.js (hunk 1 above)
Comment (AlexBuchko) [reply frontend/src/AboutPage.js:9]: Thanks, I'll delete that!

File: frontend/src/AboutPage.js [hunk 2]
```diff
@@ -9,9 +9,10 @@ function AboutPage() {
 
     useEffect(() => {
         getFood().then((result) => {
+            //@ts-ignore
```
Comment (Daniel-Redfern) [inline frontend/src/AboutPage.js:12]: Is this comment needed? I have no idea what this does

File: frontend/src/AboutPageHelper.js [hunk 3]
```diff
@@ -3,9 +3,12 @@ import NutritionTable from "./NutritionTable";
 import Reviews from "./Reviews";
 import { Link } from "react-router-dom";
 
+import "./About.css";
+
 function AboutPageHelper(props) {
     const { food, setFood } = props;
     let animalProductsMessage;
+
```
Comment (Daniel-Redfern) [inline frontend/src/AboutPageHelper.js:11]: Random Line added here, is this a prettier change?

**Review by Daniel-Redfern — COMMENTED** [review]
Looks good to me!

File: frontend/src/AboutPage.js (hunk 2 above)
Comment (AlexBuchko) [reply frontend/src/AboutPage.js:12]: thanks, I'll add an explanation for it

File: frontend/src/AboutPageHelper.js (hunk 3 above)
Comment (AlexBuchko) [reply frontend/src/AboutPageHelper.js:11]: That was just to separate between the variable declarations and the rest of the code

**Review by Ximixiyou — COMMENTED** [review]
looks good to me, good job

**Pull request merged**